lib/webpacker/railtie.rb

```
@@ -8,7 +8,7 @@ class Webpacker::Engine < ::Rails::Engine
   config.webpacker = ActiveSupport::OrderedOptions.new
 
   initializer "webpacker.proxy" do |app|
-    insert_middleware = Webpacker.config.dev_server.present? rescue nil
```

There is no need to rescue and swallow errors here.

---

Don't think the local variable explains anything. Should just inline into the conditional.

---

Agree.